nodejs-example/README.md

```diff
@@ -1,40 +1,5 @@
 # Kotlin/Wasm Node.js example
 
-This example showcases a simple application printing "Hello World" in Node.js.
-
-![](screenshots/hello-world.png)
-
-> **Note:**
-> This application uses a nightly version of Node.js 20.
-
-Follow the instructions in the sections below to try out this Kotlin/Wasm application.
-
-## Set up the environment
-
-Before starting, ensure you have the necessary IDE setup to run the application.
-
-### IDE
-
-We recommend using [IntelliJ IDEA 2023.1 or later](https://www.jetbrains.com/idea/) to work with the project.
-It supports Kotlin/Wasm out of the box.
-
-## Build and run
-
-To build and run the application:
-
-1. In IntelliJ IDEA, open the repository.
-2. Navigate to the `nodejs-example` project folder.
-3. Run the application by typing the following Gradle command in the terminal:
-
-   `./gradlew wasmJsNodeRun`
-
-## Feedback and questions
-
-Give it a try and share your feedback or questions in our [#webassembly](https://slack-chats.kotlinlang.org/c/webassembly) Slack channel.
-[Get a Slack invite](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up).
-You can also share your comments with [@bashorov](https://twitter.com/bashorov) on X (Twitter).
-
-## Learn more
-
-* [Kotlin/Wasm](https://kotl.in/wasm/)
-* [Other Kotlin/Wasm examples](https://github.com/Kotlin/kotlin-wasm-examples/tree/main)
\ No newline at end of file
+> [!IMPORTANT]  
+> The example was moved to a separate repository as a template so that you can easily clone it and play around
+> You can find the code of the example here: [Kotlin/kotlin-wasm-nodejs-template](https://github.com/Kotlin/kotlin-wasm-nodejs-template)
```

Comment:
```suggestion
> The example was moved to a separate repository as a template so that you can easily clone it and play around.
>
> You can find the code of the example here: [Kotlin/kotlin-wasm-nodejs-template](https://github.com/Kotlin/kotlin-wasm-nodejs-template).
```